### PR TITLE
API documentation for Repository Update feature

### PR DESCRIPTION
**Pull request by cinlloc**
Add API and DTO documentation for _Repository Update_ new feature.
Cf. internal User Story [#6719 ](https://projects.usharesoft.com/issues/6719)_As an admin, I want to manage repository updates_, RFE [#6999](https://projects.usharesoft.com/issues/6999).

### CODE REVIEW

File: apis/en/source/index.rst [hunk 1]
```diff
@@ -34,6 +34,7 @@ This documentation provides a description of UForge AppCenter RESTful interface.
     pages/organization/overview
     pages/project/overview
     pages/repositories/overview
+    pages/repositoryupdate/overview
```
Comment (jgweir) [inline apis/en/source/index.rst]: This should not be a new category but inside the "repositories" category

File: apis/en/source/pages/dto-objects/Repository.rst [hunk 2]
```diff
@@ -18,8 +18,8 @@ The list of attributes for ``repository`` are:
 	* ``distributionsUri`` (anyURI): the uri resource to retrieve the operating system(s) of this repository (may contain minor versions)
 	* ``url`` (string): the URL endpoint of this repository
 	* ``packagingType``: the package type ths repository holds
-	* ``lastUpdated`` (dateTime): the date this repository last recieved a package update
```
Comment (jgweir) [inline apis/en/source/pages/dto-objects/Repository.rst]: spelling mistake: "received"

File: apis/en/source/pages/dto-objects/Repository.rst [hunk 3]
```diff
@@ -18,8 +18,8 @@ The list of attributes for ``repository`` are:
...
-	* ``status`` (:ref:`status-object`): the current status of the repository
+	* ``lastUpdated`` (dateTime): the date this repository last received a package update
+	* ``repositoryUpdates``: the list of update for the repository (see :ref:`repositoryUpdate-object`
```
Comment (jgweir) [inline apis/en/source/pages/dto-objects/Repository.rst]: needs to be plural: "updates for...."

File: apis/en/source/pages/dto-objects/RepositoryUpdate.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.. Copyright 2017 FUJITSU LIMITED
+
+.. _repositoryupdate-object:
+
+repositoryUpdate
+================
+
+Holds information used by an update of a :ref:`repository-object`
```
Comment (jgweir) [inline apis/en/source/pages/dto-objects/RepositoryUpdate.rst]: Sorry but this sentence doesn't make sense.

Is it not something like: "Information used to force a repository to search for package updates"

File: apis/en/source/pages/dto-objects/Repository.rst (hunk 2 above)
Comment (cinlloc) [inline apis/en/source/pages/dto-objects/Repository.rst]: This line was modified, so it's ok in new version.

File: apis/en/source/index.rst (hunk 1 above)
Comment (cinlloc) [inline apis/en/source/index.rst]: Ok.

File: apis/en/source/pages/dto-objects/Repository.rst (hunk 3 above)
Comment (cinlloc) [inline apis/en/source/pages/dto-objects/Repository.rst]: Ok.

File: apis/en/source/pages/dto-objects/RepositoryUpdate.rst (hunk 4 above)
Comment (cinlloc) [inline apis/en/source/pages/dto-objects/RepositoryUpdate.rst]: What about: `Information about a repository package update` or `Information about a package update of a repository`?

File: apis/en/source/index.rst (hunk 1 above)
Comment (cinlloc) [inline apis/en/source/index.rst]: In fact, this have an impact on source code (need to move all methods to repository resource class). Need to see how to handle it.

File: apis/en/source/pages/dto-objects/RepositoryUpdate.rst (hunk 4 above)
Comment (emuus) [inline apis/en/source/pages/dto-objects/RepositoryUpdate.rst]: I would prefer your first option as it is quite clear : Information about a repository package update

**Pull request merged**